.eslintrc.json

```diff
@@ -5,6 +5,9 @@
   },
   "extends": ["airbnb-base", "plugin:jest/recommended", "plugin:security/recommended", "plugin:prettier/recommended"],
   "plugins": ["jest", "security", "prettier"],
+  "prettier/prettier": ["error", {
+   "endOfLine":"auto"
```

Comment:
why you add these code here? That makes .eslintrc.json invalid

Reply:
Otherwise i get the error which i mentioned in the slack channel

Reply:
`  "rules": {
    "no-console": "error",
    "func-names": "off",
    "no-underscore-dangle": "off",
    "consistent-return": "off",
    "jest/expect-expect": "off",
    "security/detect-object-injection": "off",
    "prettier/prettier": ["error", {
      "endOfLine":"auto"
    }]
  }`
i think you could add these in rules, otherwise it disabled eslintrc.json.